frontend/src/components/comments/CommentUpdateForm.jsx

```diff
@@ -0,0 +1,32 @@
+// NPM Packages
+import React, { useState } from "react";
+
+export default function CommentUpdateForm({ onSubmit, comment }) {
+  // Local State
+  const [contentText, setContentText] = useState(comment.contentText);
+
+  // Methods
+  const handleSubmit = () => {
+    // Invoke the passed in event callback
+    onSubmit({ contentText: contentText });
+
+    // Clear the input field
+    setContentText("");
+  };
+
+  return (
+    <form>
+      <div>
+        <div>
+          <textarea
+            value={contentText}
+            onChange={(e) => setContentText(e.target.value)}
```

Comment:
will this update the contentText for every time a change occurs? If so is it at all possible to update it only on submit?

Reply:
added as an enhancement issue